File: java/src/main/be/steinsomers/bron_kerbosch/BronKerbosch2_gpx.java
package be.steinsomers.bron_kerbosch;

import java.util.stream.Stream;

public final class BronKerbosch2_gpx implements BronKerboschAlgorithm {
    @Override
    public Stream<int[]> explore(UndirectedGraph graph) {
        return BronKerboschPivot.explore(graph, PivotChoice.MaxDegreeLocalX);
    }
}

File: kotlin/src/main/be/steinsomers/bron_kerbosch/BronKerbosch2gpx.kt
package be.steinsomers.bron_kerbosch

class BronKerbosch2gpx : BronKerboschAlgorithm {
    override fun explore(graph: UndirectedGraph, cliqueConsumer: (IntArray) -> Unit) {
        BronKerboschPivot.explore(graph, cliqueConsumer, PivotChoice.MaxDegreeLocalX)
    }
}
